### ISSUE TITLE
Contact Question

### ISSUE THREAD

**Issue by sethborne** [RecursiveThinking/recursive_thinking_website_tech_docs#1]
For getting on the Slack, Recursive Thinking Github and Zeplin - is there contact information to attach to this?  Or should this just be done at a DevDay?

**Comment by MynockSpit** [comment]
Good question. @amnevins may have an opinion, but I think it should be at DevDay.

**Comment by sethborne** [comment]
Yeah - thats what I thought - if we do it at DevDay we can verify people.  I tried signing up for a recursivethinking@gmail.com account, but its already been taken.  :(

^-- because I figured we didn't want to give out our emails.  :)

**Comment by MynockSpit** [comment]
Yeah. Also, I believe own the domain, so theoretically we could get anything@seattledevday.com (or w/e that domain Austin has is.)

**Comment by sethborne** [comment]
So I guess the question is @amnevins is there an aggregate email address for DevDay that someone may or may not check?

Otherwise, I'll put in the website to contact the people who look like they are in charge at DevDay...lets see where that goes... :)
